src/genome.rs

```diff
@@ -4538,4 +5019,1648 @@ mod tests {
+                        ("GT_CONF".to_string(), vec!["2.05".to_string()]),
+                    ]),
+                    is_filter_pass: true,
+                },
+                vcf_idx: Some(1),
+                indel_length: -91,
+                indel_nucleotides: Some("aaaaaaaaccccccccccggggggggggttttttttttaaaaaaaaaaccccccccccggggggggggttttttttttaaaaaaaaaaccc".to_string()),
+                gene_position: Some(-1),
+                codon_idx: None,
+                gene_name: Some("A".to_string()),
+            },
+        ];
+
+        for (idx, variant) in diff.variants.iter().enumerate() {
+            assert_eq!(variant, &expected_genome_variants[idx]);
+        }
+
+        assert_eq!(diff.minor_variants.len(), 0);
+
+        // This deletion should cover large deletions in A and B, as well as partial deletions in C
+        let expected_a_mutations = vec![
+            Mutation {
+                mutation: "-1_del_aaaaaaaaccccccccccgggggggggg".to_string(),
+                gene: "A".to_string(),
+                evidence: vec![Evidence {
+                    cov: Some(3),
+                    frs: Some(OrderedFloat(0.75)),
+                    genotype: "1/1".to_string(),
+                    call_type: AltType::DEL,
+                    vcf_row: VCFRow {
+                        position: 2,
+                        reference: "aaaaaaaaaccccccccccggggggggggttttttttttaaaaaaaaaaccccccccccggggggggggttttttttttaaaaaaaaaaccc".to_string(),
+                        alternative: vec!["a".to_string()],
+                        filter: vec!["PASS".to_string()],
+                        fields: HashMap::from([
+                            ("GT".to_string(), vec!["1/1".to_string()]),
+                            ("DP".to_string(), vec!["4".to_string()]),
+                            ("COV".to_string(), vec!["1".to_string(), "3".to_string()]),
+                            ("GT_CONF".to_string(), vec!["2.05".to_string()]),
+                        ]),
+                        is_filter_pass: true,
+                    },
+                    reference: "a".to_string(),
+                    alt: "aaaaaaaaccccccccccggggggggggttttttttttaaaaaaaaaaccccccccccggggggggggttttttttttaaaaaaaaaaccc".to_string(),
+                    genome_index: 3,
+                    is_minor: false,
+                    vcf_idx: Some(1),
+                }],
+                gene_position: Some(-1),
+                codes_protein: Some(false),
+                ref_nucleotides: None,
+                alt_nucleotides: None,
+                nucleotide_number: Some(-1),
+                nucleotide_index: Some(3),
+                indel_length: Some(-28),
+                indel_nucleotides: Some("aaaaaaaaccccccccccgggggggggg".to_string()),
+                amino_acid_number: None,
+                amino_acid_sequence: None,
+            },
+            Mutation {
+                mutation: "del_0.93".to_string(),
```

Comment:
I don't quite see where this comes from or what it means

Reply:
That's deletion of 93% of the gene - special representation for large deletions, documented in the `piezo` BNF for GARC